Linux/tree-common/init

```diff
@@ -183,9 +183,7 @@ signal_state kernel-started
 # Adjust time
 if [ "$(get_any NO_NTPDATE)" != "1" ]; then
     log_begin_msg "Adjusting time (ntp)"
-    NTPSRV=$(cat /etc/ntpsrv)
-    run ntpdate -d -b -p 2 "${NTPSRV}"
-    run ntpdate -d -b -p 6 "${NTPSRV}" &
+    run ntpdate -b $(cat /etc/ntpsrv)
```

Comment:
looks good

does this command have a timeout ?
may it block the full server boot in case of an issue when contacting the NTP servers ?